src/components/NotificationStack/index.js

```diff
@@ -43,13 +37,11 @@ class NotificationStack extends Component {
   }
 
   closeNotification = notificationId => {
-    this.setState(({ notifications }) => ({
-      notifications: notifications.filter(({ id }) => id !== notificationId)
-    }))
+    this.props.hideNotification(notificationId)
   }
 
   render () {
-    const { notifications } = this.state
+    const { notifications } = this.props
 
     return (
       <TransitionGroup className={styles.notificationStack}>
```

Comment:
Why do we need to have var "i", also we used the full name for this variable ("index") @devanshu-santiment

Reply:
@Partysun the index here ("i") is used to calculate the position on the y axis for each element.
Should I find an alternate way of just renaming it to `index` will work ?

Reply:
@devanshu-santiment rename it for faster readability.